bin/sensei.js

```diff
@@ -40,14 +34,11 @@ async function cli(args) {
   }
 }
 
-function serve(options) {
-  console.log("Start dev server");
```

Comment:
No need for this as webpack dev server already output everything you need to know.